config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml

```diff
@@ -407,10 +407,10 @@ presubmits:
           value: '{"AllAlpha":true,"AllBeta":true,"EventedPLEG":false}'
         - name: RUNTIME_CONFIG
           value: '{"api/all":"true"}'
-        - name: FOCUS
-          value: "."
+        - name: LABEL_FILTER
+          value: !Slow && !Disruptive && !Flaky && Feature: isSubsetOf { Alpha, Beta }
         - name: SKIP
-          value: \[Slow\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|PodSecurityPolicy|LoadBalancer|load.balancer|In-tree.Volumes.\[Driver:.nfs\]|PersistentVolumes.NFS|Network.should.set.TCP.CLOSE_WAIT.timeout|Simple.pod.should.support.exec.through.an.HTTP.proxy|subPath.should.support.existing|should.provide.basic.identity
+          value: PodSecurityPolicy|LoadBalancer|load.balancer|In-tree.Volumes.\[Driver:.nfs\]|PersistentVolumes.NFS|Network.should.set.TCP.CLOSE_WAIT.timeout|Simple.pod.should.support.exec.through.an.HTTP.proxy|subPath.should.support.existing|should.provide.basic.identity
```

Comment:
One small difference is that tests with just alpha or beta feature gate
dependency are allowed to run. However, we currently don't have any such
test because any such test also has to add a Feature:<feature gate name>
label to be skipped in normal jobs.

This runs 2322 tests with the old and new flags.